x-pack/plugins/security_solution/public/resolver/view/submenu.tsx

```diff
@@ -101,26 +101,24 @@ const NodeSubMenuComponents = React.memo(
   ({
     count,
     buttonBorderColor,
-    menuTitle,
     menuAction,
-    optionsWithActions,
     className,
     projectionMatrix,
     nodeID,
+    relatedEventStats,
   }: {
-    menuTitle: string;
     className?: string;
     menuAction?: () => unknown;
     buttonBorderColor: ButtonColor;
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
What's the reason for this?

Reply:
this prop is used exclusively by the styled component https://github.com/elastic/kibana/pull/76602/files#diff-dfeae0173694e4cf239f3a72a4d24ce1R307